harvester/actor.py

```diff
@@ -80,14 +130,140 @@ def has_person_data(self):
         """
         return self._person_dict is not None
 
+    @property
+    def _organization_uri(self):
+        """
+        Return URI for the affiliation of this actor.
+
+        The URIs are from http://uri.suomi.fi/codelist/fairdata/organization/code. For
+        FIN-CLARIAH affiliations, the home organization from the `departmentName` field
+        is used when determining the URI.
+
+        Organization names seem to always available in English, so English names are
+        used for matching.
+
+        Raises UnknownOrganizationException if URI match is not found.
+        """
+
+        organization_name = self.data["affiliation"]["organizationName_en"]
+
+        if organization_name == "FIN-CLARIN":
+            organization_name = self.data["affiliation"]["departmentName_en"]
+
+        url_base = "http://uri.suomi.fi/codelist/fairdata/organization/code"
+        organization_codes = {
```

Comment:
Mayybe from a maintainability POV this and maybe some other things from elsewhere in the codebase would benefit from being in a configuration file? Depends on how much this is likely to really be maintained in the future.

Reply:
Yea, and it would be best if this was synced with [language variant handling for titles and descriptions](https://github.com/CSCfi/Kielipankki-Metax-bridge/blob/main/harvester/metadata_parser.py#L29), but that seemed like a bigger renovation project. So I thought that it might be wisest to only work on that if it started to look like we will touch the codebase anyway. Because at the moment my money's on active maintenance not being too likely.